featurecontrol/featurecontrol.go

```diff
@@ -23,40 +23,40 @@ import (
 )
 
 const (
-	fcReceiverNameInMetrics  = "receiver-name-in-metrics"
-	fcClassicMatchersParsing = "classic-matchers-parsing"
-	fcUTF8MatchersParsing    = "utf8-matchers-parsing"
+	FeatureReceiverNameInMetrics = "receiver-name-in-metrics"
+	FeatureClassicMatchers       = "classic-matchers"
```

Comment:
Is changing the actual feature _value_ that appears in configuration considered a breaking change? Won't any configurations that had the old toggle start to crash with `Unknown option '%s' for --enable-feature`?

Reply:
Shouldn't happen, we haven't made any public releases of Alertmanager with the old feature flags.